nestkernel/archiving_node.cpp

```diff
@@ -205,20 +245,63 @@ nest::Archiving_Node::set_status( const DictionaryDatum& d )
   // We need to preserve values in case invalid values are set
   double_t new_tau_minus = tau_minus_;
   double_t new_tau_minus_triplet = tau_minus_triplet_;
+  double_t new_tau_Ca = tau_Ca_;
+  double_t new_beta_Ca = beta_Ca_;
   updateValue< double_t >( d, names::tau_minus, new_tau_minus );
   updateValue< double_t >( d, names::tau_minus_triplet, new_tau_minus_triplet );
+  updateValue< double_t >( d, names::tau_Ca, new_tau_Ca );
+  updateValue< double_t >( d, names::beta_Ca, new_beta_Ca );
 
-  if ( new_tau_minus <= 0 || new_tau_minus_triplet <= 0 )
+  if ( new_tau_minus <= 0.0 || new_tau_minus_triplet <= 0.0 )
+  {
     throw BadProperty( "All time constants must be strictly positive." );
+  }
 
   tau_minus_ = new_tau_minus;
   tau_minus_triplet_ = new_tau_minus_triplet;
 
+  if ( new_tau_Ca <= 0.0 )
+  {
+    throw BadProperty( "All time constants must be strictly positive." );
+  }
+  tau_Ca_ = new_tau_Ca;
+
+  if ( new_beta_Ca <= 0.0 )
+  {
+    throw BadProperty(
+      "For Ca to function as an integrator of the electrical activity, beta_ca needs to be greater "
+      "than 0." );
+  }
+  beta_Ca_ = new_beta_Ca;
+
   // check, if to clear spike history and K_minus
   bool clear = false;
   updateValue< bool >( d, names::clear, clear );
   if ( clear )
+  {
     clear_history();
+  }
+
+  if ( not d->known( "synaptic_elements" ) )
```

Comment:
See my comment above. Make "synaptic_elements" a name. If a string is used more than once, make it a name to avoid misspellings.